<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portfolio Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>app-icon.png</normaloff>app-icon.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>2</number>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>120</y>
     <width>921</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="chartLayout"/>
  </widget>
  <widget class="QTableView" name="ptfTableView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>741</width>
     <height>821</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Portfolio USD Value: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ptf_usd_value_QLbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>820</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>STATISTICS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>860</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10 Minutes Moving Average:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ptf_10min_mva_QLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>860</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Updating...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mva_alarm_QPButton">
   <property name="geometry">
    <rect>
     <x>1669</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>MVA Alarms</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
